classes/controller/AdminController.php

```diff
@@ -1989,10 +1989,12 @@ private function getTabs($parentId = 0, $level = 0)
                 unset($tabs[$index]);
             }
 
-            foreach ($tabs[$index]['sub_tabs'] as $sub_tab) {
-                if ((int)$sub_tab['current'] == true) {
-                    $tabs[$index]['current'] = true;
-                    $tabs[$index]['current_level'] = $sub_tab['current_level'];
+            if (!empty($tabs[$index]) && !empty($tabs[$index]['sub_tabs'])) {
```

Comment:
I was thinking the same thing, but then you will have to do

```php
if (array_key_exists($tabs,$index) && !empty($tabs[$index]) && ...)
```

it's not worth for the same result, isn't it ? :)